lib/scrabble.rb

```diff
@@ -1,7 +1,23 @@
 class Scrabble
 
   def score(word)
-    1
+    return 0 if word.nil?
+
+    ones = [1] * word.length
+    score_with_multipliers(word, ones)
+  end
```

Comment:
I really like that you built your score to use score_with_multipliers. Great strategy for keeping your code DRY.